var/spack/repos/builtin/packages/groff/package.py

```diff
@@ -0,0 +1,62 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Groff(AutotoolsPackage):
+    """Groff (GNU troff) is a typesetting system that reads
+    plain text mixed with formatting commands and produces
+    formatted output. Output may be PostScript or PDF, html, or
+    ASCII/UTF8 for display at the terminal."""
+
+    homepage = "https://www.gnu.org/software/groff/"
+    url      = "http://ftp.gnu.org/gnu/groff/groff-1.22.3.tar.gz"
+
+    # TODO: add html variant, spack doesn't have netpbm and its too
+    # complicated for me to find out at this point in time.
+    # See brew scripts for groff for guidance:
+    # https://github.com/Homebrew/homebrew-core/blob/master/Formula/groff.rb
+    # Seems troublesome...netpbm requires groff?
+    variant('pdf', default=True, description='Build the `gropdf` executable.')
+
+    depends_on('binutils')
+    depends_on('coreutils')
+    depends_on('gawk')
+    depends_on('gmake')
+    depends_on('sed')
+    depends_on('ghostscript', when='+pdf')
+
+    version('1.22.3', 'cc825fa64bc7306a885f2fb2268d3ec5')
+
+    # https://savannah.gnu.org/bugs/index.php?43581
+    # NOTE: always patch, even if not +pdf
+    def patch(self):
+        patch('gropdf.patch')
```

Comment:
Have you run this?  The patch doesn't appear to work as promised, every now and then I still get a build error.

I'd rather just tell `spack` not to build in parallel.  Does that make more sense?